t/Net/DRI/Protocol/EPP/Extensions/FCCN.t

```diff
@@ -64,15 +64,15 @@ $cs->set($dri->local_object('contact')->srid('c112574'),'tech');
 
 $R2='';
 $rc=$dri->domain_renew('example.pt',{duration => DateTime::Duration->new(years=>5), current_expiration => DateTime->new(year=>2008,month=>4,day=>3),auto_renew=>'no',not_renew=>0}); # test xml_parse_auto_renew();
-is($R1,$E1.'<command><renew><domain:renew xmlns:domain="urn:ietf:params:xml:ns:domain-1.0" xsi:schemaLocation="urn:ietf:params:xml:ns:domain-1.0 domain-1.0.xsd"><domain:name>example.pt</domain:name><domain:curExpDate>2008-04-03</domain:curExpDate><domain:period unit="y">5</domain:period></domain:renew></renew><extension><ptdomain:renew xmlns:ptdomain="http://eppdev.dns.pt/schemas/ptdomain-1.0" xsi:schemaLocation="http://eppdev.dns.pt/schemas/ptdomain-1.0 ptdomain-1.0.xsd"><ptdomain:roid/><ptdomain:autoRenew>false</ptdomain:autoRenew><ptdomain:notRenew>false</ptdomain:notRenew></ptdomain:renew></extension><clTRID>ABC-12345</clTRID></command>'.$E2,'domain_renew build');
+is_string($R1,$E1.'<command><renew><domain:renew xmlns:domain="urn:ietf:params:xml:ns:domain-1.0" xsi:schemaLocation="urn:ietf:params:xml:ns:domain-1.0 domain-1.0.xsd"><domain:name>example.pt</domain:name><domain:curExpDate>2008-04-03</domain:curExpDate><domain:period unit="y">5</domain:period></domain:renew></renew><extension><ptdomain:renew xmlns:ptdomain="http://eppdev.dns.pt/schemas/ptdomain-1.0" xsi:schemaLocation="http://eppdev.dns.pt/schemas/ptdomain-1.0 ptdomain-1.0.xsd"><ptdomain:roid/><ptdomain:autoRenew>false</ptdomain:autoRenew><ptdomain:notRenew>false</ptdomain:notRenew></ptdomain:renew></extension><clTRID>ABC-12345</clTRID></command>'.$E2,'domain_renew build');
 
 $R2='';
 $rc=$dri->domain_renounce('telepac2.pt');
-is($R1,$E1.'<command><transfer op="request"><domain:transfer xmlns:domain="urn:ietf:params:xml:ns:domain-1.0" xsi:schemaLocation="urn:ietf:params:xml:ns:domain-1.0 domain-1.0.xsd"><domain:name>telepac2.pt</domain:name></domain:transfer></transfer><extension><ptdomain:transfer xmlns:ptdomain="http://eppdev.dns.pt/schemas/ptdomain-1.0" xsi:schemaLocation="http://eppdev.dns.pt/schemas/ptdomain-1.0 ptdomain-1.0.xsd"><ptdomain:roid/><ptdomain:renounce>true</ptdomain:renounce></ptdomain:transfer></extension><clTRID>ABC-12345</clTRID></command>'.$E2,'domain_renounce build');
+is_string($R1,$E1.'<command><transfer op="request"><domain:transfer xmlns:domain="urn:ietf:params:xml:ns:domain-1.0" xsi:schemaLocation="urn:ietf:params:xml:ns:domain-1.0 domain-1.0.xsd"><domain:name>telepac2.pt</domain:name></domain:transfer></transfer><extension><ptdomain:renounce xmlns:ptdomain="http://eppdev.dns.pt/schemas/ptdomain-1.0" xsi:schemaLocation="http://eppdev.dns.pt/schemas/ptdomain-1.0 ptdomain-1.0.xsd"><ptdomain:roid/><ptdomain:renounce>true</ptdomain:renounce></ptdomain:renounce></extension><clTRID>ABC-12345</clTRID></command>'.$E2,'domain_renounce build');
```

Comment:
This looks like a fix (extension change) (and see the module itself). I have therefor accepted the change, even though I don't know for sure. I think there is some unmerged work on .pt somewhere too.